service/api_patch.go

```diff
@@ -470,7 +282,7 @@ func (as *APIServer) existingPatchRequest(w http.ResponseWriter, r *http.Request
 			return
 		}
 		p.PatchedConfig = string(projectYamlBytes)
-		_, err = model.FinalizePatch(p, &as.Settings)
+		_, err = model.FinalizePatch(p, as.Settings.Credentials["github"])
```

Comment:
see lower comment about writing an accessor method for getting the github token.

Reply:
done